### PR TITLE
Adding support for log4shell

### PR DESCRIPTION
**Pull request by jezd-axyl**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] I have read the [CONTRIBUTING](https://github.com/fabasoad/pre-commit-snyk/blob/main/CONTRIBUTING.md)
      doc.
- [x] Tests for the changes have been added (for bug fixes / features).
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features).

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the new behavior

- Added support for testing for log4j
-
-

## Does this introduce a breaking change

- [ ] Yes
- [x] No

## Other information

---

Closes #{IssueNumber}

### CODE REVIEW

File: .pre-commit-hooks.yaml [hunk 1]
```diff
@@ -21,3 +21,10 @@
   entry: hooks/snyk-test.sh
   language: script
   pass_filenames: false
+
+- id: snyk-log4shell
+  name: Snyk log4shell
+  description: Runs 'snyk log4shell test' command
```
Comment (fabasoad) [inline .pre-commit-hooks.yaml:27]:
```suggestion
  description: Runs 'snyk log4shell' command
```

**Review by fabasoad — CHANGES_REQUESTED** [review]
Thank you for your contribution! One small change and we are good to go 🚀

File: .pre-commit-hooks.yaml (hunk 1 above)
Comment (jezd-axyl) [reply .pre-commit-hooks.yaml]: Done

**Review by fabasoad — APPROVED** [review]

**Comment by fabasoad** [comment]
@jezd-axyl Thank you! Merging it.

**Pull request merged**